### PR TITLE
Add WinMLExplorer to sample code

### PR DESCRIPTION
**Pull request by simplywilson**

### CODE REVIEW

File: Samples/UWP/WinMLExplorer/WinMLExplorer/Models/ClassifierModel.cs [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+                DateTime startTime = DateTime.UtcNow;
+
+                // Evaluate model
+                //LearningModelEvaluationResultPreview evaluationResult = await this.Model.EvaluateAsync(binding, result.CorrelationId);
+
+                Pcb_x002D_aoi_x002D_genimage_x002D_v2ModelOutput output = await m.EvaluateAsync(input);
+
+                duration = DateTime.UtcNow - startTime;
+
+                //using (SoftwareBitmap bitmapBuffer = new SoftwareBitmap(BitmapPixelFormat.Bgra8,
```
[the remark this answers is not in the document]
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/Models/ClassifierModel.cs]: Thanks. This is part of an old file. It is now deleted

File: Samples/UWP/WinMLExplorer/WinMLExplorer/Controls/CameraControl.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        {
+            if (this.CaptureManager.CameraStreamState != Windows.Media.Devices.CameraStreamState.Streaming
+                || !this.FrameProcessingSemaphore.Wait(0))
+            {
+                return;
+            }
+
+            try
+            {
+                const BitmapPixelFormat InputPixelFormat = BitmapPixelFormat.Nv12;
```
[the remark this answers is not in the document]
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/Controls/CameraControl.xaml.cs]: Fixed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        {
+            MLModelResult result = new MLModelResult()
+            {
+                CorrelationId = Guid.NewGuid().ToString()
+            };
+
+            try
+            {
+                // Set duration
+                TimeSpan duration = new TimeSpan();
```
[the remark this answers is not in the document]
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: Fixed.

File: Samples/UWP/WinMLExplorer/WinMLExplorer/Package.appxmanifest [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    <Application Id="App" Executable="$targetnametoken$.exe" EntryPoint="WinMLExplorer.App">
+      <uap:VisualElements DisplayName="Windows Machine Learning Explorer" Square150x150Logo="Assets\Square150x150Logo.png" Square44x44Logo="Assets\Square44x44Logo.png" Description="Windows Machine Learning Explorer Sample App" BackgroundColor="#EBEBEB">
+        <uap:SplashScreen Image="Assets\SplashScreen.png" BackgroundColor="#EBEBEB" />
+      </uap:VisualElements>
+    </Application>
+  </Applications>
+  <Capabilities>
+    <Capability Name="internetClient" />
+    <uap:Capability Name="picturesLibrary" />
+    <DeviceCapability Name="microphone" />
```
[the remark this answers is not in the document]
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/Package.appxmanifest]: Won't fix for this one. I agree with what you said, but for some reason, not enabling the microphone, my web cam does not show the picture.

File: Samples/UWP/WinMLExplorer/WinMLExplorer/ViewModels/MainViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+            : base()
+        {
+            // Initialize models
+            this.Models = new List<WinMLModel>()
+            {
+                new PcbModel(),
+            };
+
+            // Initialize cameras
+            this.CameraNames = Task.Run(async () => await DeviceUtility.GetAvailableCameraNamesAsync()).Result.ToList();
```
[the remark this answers is not in the document]
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/ViewModels/MainViewModel.cs]: Fixed.

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/PcbModel.cs [hunk 6]
```diff
@@ -53,12 +53,16 @@ public sealed class PcbModel : WinMLModel
 
         protected override async Task EvaluateAsync(MLModelResult result, VideoFrame inputFrame)
         {
+            // Initialize the input
             PcbModelInput input = new PcbModelInput() { data = inputFrame };
 
+            // Evaludate the input
```
Comment (LPBourret) [inline Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/PcbModel.cs]: typo: Evaluate

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs [hunk 7]
```diff
@@ -32,18 +44,40 @@ public WinMLModel()
             Task.Run(async () => await Initialize()).Wait();
         }
 
+        /// <summary>
+        /// Name of the input (ie: circuit board) to be display on the main ui
```
Comment (LPBourret) [inline Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: typo: displayed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs [hunk 8]
```diff
@@ -32,18 +44,40 @@ public WinMLModel()
...
+        /// </summary>
         public abstract string DisplayInputName { get; }
 
+        /// <summary>
+        /// Mininum probability to be displayed on the result list
```
Comment (LPBourret) [inline Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: typo: Minimum

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs [hunk 9]
```diff
@@ -99,11 +139,17 @@ public WinMLModel()
...
+        /// Evaluate input video frame work ml model result
+        /// </summary>
         protected virtual async Task EvaluateAsync(MLModelResult result, VideoFrame inputFrame)
         {
             throw new NotImplementedException();
         }
-        
+
+        /// <summary>
+        /// Initilize the ML model
```
Comment (LPBourret) [inline Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: typo: Initialize

File: Samples/UWP/WinMLExplorer/WinMLExplorer/ViewModels/MainViewModel.cs [hunk 10]
```diff
@@ -144,6 +181,9 @@ public string LandingMessage
             }
         }
 
+        /// <summary>
+        /// Set the results to be display after evaluation
```
Comment (LPBourret) [inline Samples/UWP/WinMLExplorer/WinMLExplorer/ViewModels/MainViewModel.cs]: typo: displayed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/PcbModel.cs (hunk 6 above)
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/PcbModel.cs]: Fixed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs (hunk 7 above)
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: Fixed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs (hunk 8 above)
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: Fixed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs (hunk 9 above)
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/MLModels/WinMLModel.cs]: Fixed

File: Samples/UWP/WinMLExplorer/WinMLExplorer/ViewModels/MainViewModel.cs (hunk 10 above)
Comment (simplywilson) [reply Samples/UWP/WinMLExplorer/WinMLExplorer/ViewModels/MainViewModel.cs]: Fixed

**Pull request merged**